c/validation/adbc_validation.cc

```diff
@@ -508,8 +508,8 @@ void ConnectionTest::TestMetadataGetObjectsDbSchemas() {
         // type: list<table_schema>
         struct ArrowArrayView* db_schema_tables_list = catalog_db_schemas->children[1];
 
-        ASSERT_FALSE(ArrowArrayViewIsNull(catalog_db_schemas_list, row))
-            << "Row " << row << " should have non-null catalog_db_schemas";
+        // ASSERT_FALSE(ArrowArrayViewIsNull(catalog_db_schemas_list, row))
```

Comment:
AFAIK postgres only allows you to retrieve the schemas from the database you are currently connected to, but we can see what other databases exist. So either this test is too restrictive being in the base tests or we should make it so that the postgres driver only lists the current database in GetObjects; I think the former option is better

Reply:
The intent is that if you request the schemas, it should come back as a (possibly empty) list, not null.

Reply:
Ah OK makes sense. Thanks for the guidance